src/tools/spark-sstfile-generator/src/main/scala/com/vesoft/nebula/tools/generator/v2/SparkClientGenerator.scala

```diff
@@ -310,7 +309,11 @@ object SparkClientGenerator {
               val values = (for {
                 property <- valueProperties if property.trim.length != 0
               } yield extraValue(row, property)).mkString(",")
-              (String.valueOf(extraValue(row, vertex)), values)
+              (if (String.valueOf(extraValue(row, vertex)).startsWith("\"") && String.valueOf(extraValue(row, vertex)).endsWith("\"")) {
```

Comment:
if the string start with quotation mark or end with quotation mark how to deal with it ?